client/blocks/simple-site-rename-form/dialog.jsx

```diff
@@ -130,7 +130,9 @@ class SiteRenamerConfirmationDialog extends PureComponent {
 						checked={ this.state.isConfirmationChecked }
 						onChange={ this.toggleConfirmationChecked }
 					/>
-					<span>{ translate( "I've double-checked and understand there is no undo." ) }</span>
+					<span>
+						{ translate( "I understand that I can't undo this change to my site address." ) }
```

Comment:
I feel like `I understand that I won't be able to undo this change to my site address.` might be more accurate, because at this point it hasn't happened yet.

Reply:
Hmm, I see what you're saying! There's context since we're in the dialog already but your suggestion may still be a touch more explicit - @benhuberman, do you have any thoughts on this?

Reply:
Sweet! Update made - I'll merge once checks are ready :)